messages.txt

```diff
@@ -0,0 +1 @@
+tseting again!
```

Comment:
You generally don't want to commit files generated at runtime to a repository- you can add `*.txt` (and `*.md`) to your `.gitignore` to avoid accidentally committing this type of file. (https://git-scm.com/docs/gitignore)

Reply:
As of right now, if the messages.txt file does not exist, when the code runs it will cause an error when it eventually attempts to read a file. Should I remove it for now, and then in the next commit, I add in supplemental code that'll create a file if there isn't one already?

Reply:
If you use `+a` instead of `+r` when opening, the file will be created if it does not already exist: https://stackoverflow.com/questions/2967194/open-in-python-does-not-create-a-file-if-it-doesnt-exist

Going forward, you might want to write a function which does your file opening/writing and you can just pass text into it. That way, your code will be simpler to read, and if you change the way you do file writing, you will only need to change it in one place!

Reply:
Alright, I will put the function into consideration. For now I've updated the code to use a+.